mmv1/products/bigquery/api.yaml

```diff
@@ -978,6 +978,40 @@ objects:
             description: |
               The geographic location of the job. The default value is US.
             default_value: 'US'
+      - !ruby/object:Api::Type::NestedObject
+        name: 'status'
+        output: true
```

Comment:
Not sure if this is needed. Because of that terraform generator doesn't set `maxItems: 1`